server/.gitignore

```diff
@@ -0,0 +1,4 @@
+.pytest_cache/
+*.pyc
+ff-env/
+__pycache__/
```

Comment:
Add a blank line at the end of this file

Reply:
Done